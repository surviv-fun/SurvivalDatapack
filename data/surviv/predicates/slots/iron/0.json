[
    {
        "condition": "minecraft:entity_properties",
        "entity": "this",
        "predicate": {
            "equipment": {
                "mainhand": {
                    "items": ["minecraft:iron_sword", "minecraft:iron_axe", "minecraft:iron_pickaxe", "minecraft:iron_shovel", "minecraft:iron_hoe"]
                }
            }
        }
    }
]
