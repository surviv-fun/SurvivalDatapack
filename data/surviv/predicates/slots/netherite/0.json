[
    {
        "condition": "minecraft:entity_properties",
        "entity": "this",
        "predicate": {
            "equipment": {
                "mainhand": {
                    "items": ["minecraft:netherite_sword", "minecraft:netherite_axe", "minecraft:netherite_pickaxe", "minecraft:netherite_shovel", "minecraft:netherite_hoe"]
                }
            }
        }
    }
]
