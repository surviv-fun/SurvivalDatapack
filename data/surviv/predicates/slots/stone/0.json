[
    {
        "condition": "minecraft:entity_properties",
        "entity": "this",
        "predicate": {
            "equipment": {
                "mainhand": {
                    "items": ["minecraft:stone_sword", "minecraft:stone_axe", "minecraft:stone_pickaxe", "minecraft:stone_shovel", "minecraft:stone_hoe"]
                }
            }
        }
    }
]
