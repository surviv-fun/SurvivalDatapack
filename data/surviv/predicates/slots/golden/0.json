[
    {
        "condition": "minecraft:entity_properties",
        "entity": "this",
        "predicate": {
            "equipment": {
                "mainhand": {
                    "items": ["minecraft:golden_sword", "minecraft:golden_axe", "minecraft:golden_pickaxe", "minecraft:golden_shovel", "minecraft:golden_hoe"]
                }
            }
        }
    }
]
